content/blog/2022/10/camel-k-cicd/index.md

```diff
@@ -0,0 +1,587 @@
+
+You can also watch the `Pod` created by the event listener to see if the `ping` has reached out your cluster:
+
+```
+k logs -n development el-github-push-674dcb5f4-blbrb -f
+```
+
+If all is good, let's keep monitoring that `Pod` and make any change to your repository. As soon as you push, you should see it flowing in your log like:
+
+```
+{"severity":"info","timestamp":"2022-10-04T11:02:30.537Z","logger":"eventlistener","caller":"sink/sink.go:409","message":"ResolvedParams : []","eventlistener":"github-push","namespace":"development","/triggers-eventid":"b7fda1d3-044e-402a-b2c2-900afd849681","eventlistenerUID":"98b1322c-1793-425a-8577-8fec3e708b3f","/triggers-eventid":"b7fda1d3-044e-402a-b2c2-900afd849681","/trigger":"github-listener"}
+{"severity":"info","timestamp":"2022-10-04T11:02:30.539Z","logger":"eventlistener","caller":"resources/create.go:98","message":"Generating resource: kind: &APIResource{Name:pipelineruns,Namespaced:true,Kind:PipelineRun,Verbs:[delete deletecollection get list patch create update watch],ShortNames:[pr prs],SingularName:pipelinerun,Categories:[tekton tekton-pipelines],Group:tekton.dev,Version:v1beta1,StorageVersionHash:RcAKAgPYYoo=,}, name: github-push-pipeline-run-"}
+{"severity":"info","timestamp":"2022-10-04T11:02:30.539Z","logger":"eventlistener","caller":"resources/create.go:106","message":"For event ID \"b7fda1d3-044e-402a-b2c2-900afd849681\" creating resource tekton.dev/v1beta1, Resource=pipelineruns"}
+```
+
+That means the `Pipeline` was triggered and in a few minutes you should have your shining new feature available in production!
+
+### Test webhook locally
+
+During the development of your pipeline (or if you want to use a local cluster such as `minikube`), you may be interested in simulating the push event. This should be as easy as running something like:
+
+```
+http post http://192.168.49.2:32698/my-cd-pipeline X-GitHub-Event:push json=test 
+```
+
+# Conclusion
+
+It was a long post, but, I am sure you appreciated the effort we've put in wrapping up all the resources required to have a full CICD experience on a Camel K integration. The possibility offered by Tekton or any other pipeline application on Kubernetes are a great complement on the experience we are providing with the `kamel promote` feature.
+
+## Caveat
+
+While working on this blog, I've spotted a problem that may [prevent you updating the Integration](https://github.com/apache/camel-k/issues/3673) via `kamel promote` on a production environment. It was promptly fixed and available by using `1.11` or `1.10` nightly releases. You can change the `Tasks` configured to use `docker.io/testcamelk/camel-k:1.11.0-nightly` instead of `docker.io/apache/camel-k:1.10.0`
```

Comment:
Newline at the end of file helps git a bit
```suggestion
While working on this blog, I've spotted a problem that may [prevent you updating the Integration](https://github.com/apache/camel-k/issues/3673) via `kamel promote` on a production environment. It was promptly fixed and available by using `1.11` or `1.10` nightly releases. You can change the `Tasks` configured to use `docker.io/testcamelk/camel-k:1.11.0-nightly` instead of `docker.io/apache/camel-k:1.10.0`

```